dockerfiles/e2e/entrypoint.sh

```diff
@@ -8,7 +8,6 @@ kill_ffmpeg(){
   wait "$ffmpeg_pid"
   mkdir -p /tmp/e2e/report/
   cp /tmp/ffmpeg_report/* /tmp/e2e/report/
```

Comment:
if there's no content in /tmp/ffmpeg_report/ will this fail with exit 1? maybe we should check if the folder exists first, eg.,

```
if [[ -d /tmp/ffmpeg_report/ ]]; then
...
fi
```

?